gemrb/includes/ie_stats.h

```diff
@@ -151,7 +151,7 @@ namespace GemRB {
 
 // can't walk or attack - helpless
 // FIXME: why is STATE_BERSERK on the list?
-#define STATE_CANTMOVE ((STATE_MINDLESS|STATE_STUNNED|_DEAD_STATES|STATE_DEACTIVATED)^(STATE_CHARMED|STATE_CONFUSED)) // 0x180fef
+#define STATE_CANTMOVE ((STATE_MINDLESS|STATE_STUNNED|_DEAD_STATES|STATE_DEACTIVATED)^(STATE_CHARMED|STATE_CONFUSED|STATE_PANIC|STATE_BERSERK)) // 0x180fef
```

Comment:
justification? Clearly one can move while panicking or berserking.

Reply:
> justification? Clearly one can move while panicking or berserking.

Yes, before the edits, the frightened skeletons did not move.

Reply:
Sorry, you're right — I misread the parentheses placement.